src/cfml/system/services/ServerService.cfc

```diff
@@ -1208,6 +1209,10 @@ component accessors="true" singleton {
 			.append( '--cookie-secure' ).append( serverInfo.sessionCookieSecure )
 			.append( '--cookie-httponly' ).append( serverInfo.sessionCookieHTTPOnly );
 			
+			if(len(trim(serverInfo.preferedBrowser))){
+				args.append( '--prefered-browser' ).append( serverInfo.preferedBrowser );
```

Comment:
This should just be sending the config setting.  Also... where is the config setting??  I don't see it anywhere in this pull!